pkg/pillar/types/ledmanagertypes_test.go

```diff
@@ -0,0 +1,32 @@
+package types
+
+import (
+	"testing"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type TestDeriveLedCounterMatrixEntry struct {
+	ledCounter         int
+	usableAddressCount int
+	expectedValue      int
+}
+
+func TestDeriveLedCounter(t *testing.T) {
+	log.Infof("TestLookupIoBundle: START\n")
+	testMatrix := []TestDeriveLedCounterMatrixEntry{
+		{ledCounter: 0, usableAddressCount: 0, expectedValue: 1},
```

Comment:
In general, I'd prefer if you could follow the style from this article: https://medium.com/agrea-technogies/basic-testing-patterns-in-go-d8501e360197 when writing Table based testing.

Do you think you can re-write this using that style?

Reply:
Would you like me to use the assert package mentioned in the article?

Reply:
testify is a good choice https://github.com/stretchr/testify